src/x/debug/debug.go

```diff
@@ -68,9 +71,44 @@ func NewZipWriter(iopts instrument.Options) ZipWriter {
 	}
 }
 
+// NewPlacementZipWriterWithDefaultSources returns a zipWriter with the following
+// debug sources already registered: CPU, heap, host, goroutines, namespace and placement info.
+func NewPlacementZipWriterWithDefaultSources(
```

Comment:
nit: `NewPlacementAndNamespaceZipWriterWithDefaultSources(...)` so it mentions namespace?